tests/Extension/StatusRuntimeTest.php

```diff
@@ -31,4 +69,53 @@ public function testStatusClassDefaultValue(): void
         $runtime->addStatusService($statusService);
         $this->assertSame('test-value', $runtime->statusClass(new \stdClass(), 'getStatus', 'test-value'));
     }
+
+    /**
+     * NEXT_MAJOR: remove this method.
+     */
+    public function testFlashManagerInSonataStatusRenderer(): void
+    {
+        // Given
+        $this->flashManager->addFlash('my_bundle_success', 'hey, success dude!');
+        $this->session->getFlashBag()->set('my_second_bundle_success', 'hey, success dude!');
+
+        // When
+        $this->flashManager->get('success');
+
+        // Then
+        $statusRuntime = new StatusRuntime();
+        $statusRuntime->addStatusService($this->flashManager);
+
+        $this->assertSame(
+            'danger',
+            $statusRuntime->statusClass($this->flashManager, 'error', 'default_class')
+        );
+
+        $this->assertSame(
+            'default_class',
+            $statusRuntime->statusClass($this->flashManager, 'blabla', 'default_class')
+        );
+    }
+
+    /**
+     * Returns a Symfony session service.
+     *
+     * @return Session
```

Comment:
Could we use real type declarations?

Reply:
Of course. I will do it tomorrow. Now i go sleep. Also i will update documentation and changelog.